resources/assets/js/components/BarChart.vue

```diff
@@ -0,0 +1,16 @@
+<script>
+  import { Bar, mixins } from 'vue-chartjs';
+  const { reactiveProp } = mixins;
+
+  export default {
+    extends: Bar,
+    props: {
+      options: {}
```

Comment:
requires `type` and `default` or `required`

Reply:
Done